client/core/core_test.go

```diff
@@ -765,15 +766,28 @@ func newTestRig() *testRig {
 
 	// Set the global waiter expiration, and start the waiter.
 	queue := wait.NewTickerQueue(time.Millisecond * 5)
-	go queue.Run(tCtx)
+	ctx, cancel := context.WithCancel(tCtx)
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		queue.Run(ctx)
+	}()
+
+	dc, conn, acct := testDexConnection(ctx)
 
-	dc, conn, acct := testDexConnection()
+	shutdown := func() {
+		cancel()
+		wg.Wait()
+		dc.connMaster.Wait()
+	}
```

Comment:
Similar changes in PR https://github.com/decred/dcrdex/pull/967.  On that work I had noticed dozens of latency queues ticking away in the later test cases.